misc/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordDirectConverter.java

```diff
@@ -71,6 +71,9 @@ public class JsonToRecordDirectConverter {
 
     private JsonToRecordDirectConverter() {}
 
+    private static final String newRecordName = "NewRecord";
+    private static final String arrayRecordSuffix = "Item";
```

Comment:
Shall we use uppercase with words separated by underscores for constants?
ex: NEW_RECORD_NAME